yt_dlp/extractor/mojevideo.py

```diff
@@ -0,0 +1,43 @@
+import re
+from .common import InfoExtractor
+from ..utils import ExtractorError
+
+
+class MojevideoIE(InfoExtractor):
+    _VALID_URL = r'https://www\.mojevideo\.sk/video/(?P<id>[0-9]+)'
+
+    _TESTS = [{
+        'url': 'https://www.mojevideo.sk/video/3d17c/chlapci_dobetonovali_sme_mame_hotovo.html',
+        'md5': '384a4628bd2bbd261c5206cf77c38c17',
+        'info_dict': {
+            'id': '250236',
+            'ext': 'mp4',
+            'title': 'Chlapci dobetónovali sme, máme hotovo!',
+            'description': 'Celodenná práca bola za pár sekúnd fuč. Betón stiekol k susedovi, kam aj zrútil celý plot, ktorý polámal aj tuje. Chlapom zostali iba oči pre plač.'
+        }
+    }]
+
+    def _real_extract(self, url):
+        webpage = self._download_webpage(url, 1)
+
+        video_id = re.search(r'vId=(\d+)', webpage).group(1)
+        video_expiration = re.search(r"vEx='(\d+)'", webpage).group(1)
+        video_hash = re.search(r'vHash=\[([^\]]+)', webpage).group(1).split(",")[0].replace("'", "")
+        video_title = re.search(r'<h1>(.*?)</h1>', webpage).group(1)
```

Comment:
- use `_html_search_regex()` when the returned value may contain html that needs to be cleaned
- metadata extraction such `title`, `description`, etc should be non-fatal. the `_search_regex` helper methods have a `fatal` kwarg and a `default` kwarg to make it simple to do so
- if the title is available from multiple sources, there should be at least 1 fallback. here we can use the helper method `_og_search_title()`
- if possible, you should inline metadata extraction in the return
 
 so we would do this
 ```py
        title = self._html_search_regex(
            r'<h1>[^<]+</h1>', webpage, 'title', default=None) or self._og_search_title(webpage, fatal=False)
```
except we're not going to do it at this point in the code, so let's remove
```suggestion
```